bundler/lib/dependabot/bundler/file_parser.rb

```diff
@@ -47,7 +50,16 @@ def ecosystem
 
       sig { returns(Ecosystem::VersionManager) }
       def package_manager
-        PackageManager.new(bundler_version)
+        @package_manager ||= PackageManager.new(bundler_raw_version)
+      end
+
+      sig { returns(T.nilable(Ecosystem::VersionManager)) }
+      def language
+        return @language if defined?(@language)
+
+        return nil unless package_manager.unsupported?
+
+        Language.new(ruby_raw_version)
```

Comment:
**Review Tip**: If the package manager is unsupported, the language version may not be retrievable. Additionally, fetching the language version is unnecessary, as the process will raise an unsupported package manager error.